.github/workflows/build.yml

```diff
@@ -49,66 +67,14 @@ jobs:
         -Dmaven.test.failure.ignore=true
         -Drt.equinox.binaries.loc=${{ github.workspace }}/rt.equinox.binaries
         clean verify
-    - name: Upload Test Results
-      uses: actions/upload-artifact@v4
-      with:
-        name: test-results
```

Comment:
@laeubi can you tell if the changing this requires adapting the test-results workflow?
I searched for the name but didn't found a match that seemed to need adoption.

Reply:
The test-result comments says everything looks good, so I assume it is fine.